### ISSUE TITLE
Invoices include 'how to pay' information - including Stripe link

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#5209]
## Description

After #5208 invoices sent after changes to orders will have the correct information about balances due.

A standard feature on a normal invoice is to include information on how the Customer can pay that invoice. This should be relatively easy to add to our invoices, so that Hubs can just send them out and Customers can pay any balance owing

Shops could add payment information to be printed on their invoices in the Business Details settings page

![image.png](https://images.zenhubusercontent.com/54b83fb37c6ccc6277a6ea46/d3fb71a6-1ad8-4304-9f83-c4afc560590c)

[on hold as I pursue whether it is actually possible to just put a custom stripe link in there .. ]
[of course would be even better if it could open a payment session to take the payment and reconcile against the account, but i reckon that's getting well out of papercut land] 

## Acceptance Criteria & Tests

1. Hub / Shop can send an updated invoice with correct balance (#5208)
2. Customer can easily pay their balance due 

NB. This proposed solution still leaves the payment unreconciled i.e. not matched against the amount owing in OFN - that will need to be done manually.

**Comment by luisramos0** [comment]
I think that adding a text field to be added to all invoices is papercut size :+1:

The hubs would have to create the text and link to stripe themselves (we could write a guide for that).

But I am not sure exactly what URL would that be: a stripe page outside OFN that allows a customer to pay the hub. There must be such a page on stripe. We have to find out.

Building a page in OFN to do that would be a page similar to "Topup my account on this hub" :heart:  that is not papercut :-( but it would be awesome and not difficult M+

**Comment by luisramos0** [comment]
![image](https://user-images.githubusercontent.com/1640378/79040094-0d0af700-7bde-11ea-876d-89871ef63e0d.png)

**Comment by kirstenalarsen** [comment]
OK ace - I will spend some time today investigating Stripe - both for this and if we could manipulate it to handle split payments

**Comment by kirstenalarsen** [comment]
![image](https://user-images.githubusercontent.com/2911176/79085955-c81dc680-7d7d-11ea-9fa7-e20080d279d6.png)

![image](https://user-images.githubusercontent.com/2911176/79085962-cfdd6b00-7d7d-11ea-98fb-3a4cae886928.png)

https://stripe.com/docs/billing/invoices/hosted
https://stripe.com/docs/billing/invoices/create-invoice

**Comment by kirstenalarsen** [comment]
urg

**Comment by Erioldoesdesign** [comment]
Love this idea by the way! Would love to monitor if enterprise-users hack it for other purposes ;)

I would have assumed stripe would have the equivalent of a [paypal.me](https://www.paypal.com/paypalme/) link... [nope](https://www.reddit.com/r/stripe/comments/9i6jt2/does_stripe_have_an_equivalent_to_a_paypalme/) looks like you have to create a new link for each invoice via stripe.
